product-service/serverless.yml

```diff
@@ -3,9 +3,50 @@ service: product-service
 provider:
   name: aws
   runtime: nodejs12.x
-  stage: dev
+  stage: ${self:custom.stage}
   region: eu-west-1
 
+  iamRoleStatements:
+    - Effect: 'Allow'
+      Action: 'sns:*'
+      Resource:
+        Ref: SNSTopic
+    - Effect: 'Allow'
+      Action: 'sqs:*'
+      Resource:
+        - "${cf:import-service-${self:custom.stage}.SQSQueueArn}"
+
+  environment:
+    SNS_ARN:
+      Ref: SNSTopic
+
+resources:
+  Resources:
+    SNSTopic:
+      Type: AWS::SNS::Topic
+      Properties:
+        TopicName: createProductTopic
+    SNSSubsciption:
+      Type: AWS::SNS::Subscription
+      Properties:
+        Endpoint: alex.asm.1997@gmail.com
```

Comment:
I'd recommend moving emails to the .env file because this is sensitive information